frameworks/helloworld/tests/test_mono_to_multi_migrate.py

```diff
@@ -0,0 +1,67 @@
+import logging
+
+import pytest
+from tests import config
+
+import sdk_install
+import sdk_plan
+import sdk_tasks
+import sdk_upgrade
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function', autouse=True)
+def configure_package(configure_security):
```

Comment:
could rename this function to be clear that it's running for each test case, and not once for the whole file